preprocess/preprocess.go

```diff
@@ -96,11 +99,85 @@ func ExpandJSON(block *gabs.Container, vm *goja.Runtime) {
 	log.Debug().Str("Processed JSON block", block.String()).Msg("")
 }
 
-func LoadElfEnv(l2path string) {
+func SearchL2ConfigEnv(dir string) (string, error) {
+	parentDir := filepath.Dir(dir)
```

Comment:
We should start from present folder itself. What if it is a project without any subdirs.

Reply:
Yeah this case failed :(